etcdmain/grpc_proxy.go

```diff
@@ -70,6 +73,9 @@ func newGRPCProxyStartCommand() *cobra.Command {
 	cmd.Flags().StringVar(&grpcProxyCert, "cert", "", "identify secure connections with etcd servers using this TLS certificate file")
 	cmd.Flags().StringVar(&grpcProxyKey, "key", "", "identify secure connections with etcd servers using this TLS key file")
 	cmd.Flags().StringVar(&grpcProxyCA, "cacert", "", "verify certificates of TLS-enabled secure etcd servers using this CA bundle")
+	cmd.Flags().StringVar(&grpcProxyAdvertiseAddr, "advertise-addr", "127.0.0.1:23790", "advertise address (must be reachable by client)")
+	cmd.Flags().DurationVar(&grpcAutoSyncDuration, "auto-sync", time.Duration(0), "interval to auto-sync etcd cluster endpoints (0 to disable)")
```

Comment:
If this is using the naming resolver, there's no need for a poll interval-- it's backed by a watch so it'll get the update as soon as it sees it.